src/shims/console/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This file is used to override the default console behavior
+ * in order to send the logs to the splunk server.
+ */
+
+
+const SPLUNK_URL = new URL('/proxy/splunk/', window.location.origin).href;
+
+
+/**
+ * Send the log message to the splunk server.
+ *
+ * @param {any} message Message to log, this can be any type.
+ * @param {"INFO"|"ERROR"|"WARN"} severity Severity of the log message.
+ */
+function logger(message, severity) {
+  fetch(SPLUNK_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body: JSON.stringify({
+      event: {
+        message,
+        severity,
+        environment: JSON.parse(localStorage.getItem('UserStore'))?.env || 'unset',
```

Comment:
I think we need to add some extra fields so the logs are more useful.
Maybe the user agent (`navigator.userAgent` or a custom string from the `navigator` api) and also some information about the user, like the username to identify the logs, as right now there's no way of telling who did what

Reply:
Here's a screenshot of the current data 
![image](https://github.com/user-attachments/assets/77231fbb-8925-4c1e-8b05-cdb7f5207975)